script/release

```diff
@@ -27,6 +27,9 @@ then
   exit 1
 fi
 
+echo "Install build and twine"
```

Comment:
We actually don't use this script for releasing...

https://github.com/home-assistant/core/blob/956ceb6c684ae16334395a6fc6e5488ea75313fa/.github/workflows/builder.yml#L76-L80

Reply:
Thanks! Updated that too. I'm not entirely sure which workflow Github will choose once this is merged, the old or new one (for the `2022.1` release).

To be safe, it might make sense to cherry-pick the prerequisite PRs:
* #65091
* #65095
* #65129

Reply:
As far as I am aware (thus not 100% certain), it will use the default branch, which is `dev`

Reply:
Let's remove `script/release`

Reply:
> As far as I am aware (thus not 100% certain), it will use the default branch, which is `dev`

That's what I remember as well. In that case, the three merged PRs above and this one would need to be cherry-picked if accepted.

Reply:
> Let's remove `script/release`

👍🏻 Done